testutil/src/main/java/org/spine3/test/NullToleranceTest.java

```diff
@@ -356,10 +392,42 @@ public Builder excludeMethod(String methodName) {
         /**
          * Adds the default value for the method argument.
          *
-         * <p> For example, if will be added the text line,
-         * that line will be passed as the method argument for the {@code String} parameter type.
-         * During the check that default value will be passed as the method argument every time,
-         * when the particular argument is not checking.
+         * <p> Since the some classes have no value by default, that method serves to add it.
```

Comment:
An important part of the flow is missed. Please describe it as discussed vocally.